demo/midgard-sdk/src/utils/deposit.ts

```diff
@@ -0,0 +1,52 @@
+import { Data, UTxO } from "@lucid-evolution/lucid";
+import { Effect } from "effect";
+import { Deposit } from "../tx-builder/index.js";
```

Comment:
Our convention so far has been importing related modules unqualified. So here, you can just import the datum and deposit UTxO:
```suggestion
import { Datum, DepositUTxO } from "@/tx-builder/user-events/datum/types.js";
```